developer-extension/src/panel/formatNumber.ts

```diff
@@ -0,0 +1,7 @@
+export function formatNumber(n: number): string {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** `1,692,964,306,392` display for timestamps seems odd to me.
what about having a specific format for those?